config-sample.yaml

```diff
@@ -163,6 +163,21 @@ keys:
 #   urls:
 #     - https://your-host.com/handler
 
+# Signal Relay
+# since v1.4.0, a more reliable, psrpc based signal relay is available
+# this gives us the ability to more reliable proxy messages between a signal server and RTC node
+# signal_relay:
+#   # when enabled, livekit will use a more reliable, psrpc based signal relay
+#   enabled: true
+#   # number of attempts for each message before giving up
+#   max_attempts: 3
+#   # amount of time to wait for RTC node to ack
+#   timeout: 100ms
+#   # amount of time to wait before retrying
+#   backoff: 500ms
+#   # number of messages to buffer before blocking/dropping
```

Comment:
would be good to mention whether this actually blocks or drops. If there is a configuration to choose between blocking vs dropping, does it make sense to point it here.

Reply:
actually i think it's safe to say messages can be dropped in either case because stream load sheds when there are slow consumers too...